### DISCUSSION TITLE
Cherax and YimMenu

### DISCUSSION THREAD

**Discussion by Bradly7412** [YimMenu/YimMenu#2795] [Q&A]
I've been using Yim for a long time. and decided today i want to Co-load with Cherax as i have had it for a while for the extra protections. 
The trouble i am having that when i load Yim First then Cherax, i am unable to load/spawn any custom vehicles using "New", "Persistent" or "XML" it just instantly fails. But personal vehicle works fine, This only fails when i have cherax loaded with Yim. i have tried to turn off all of the protections in Cherax but still no vehicle spawning, Just wondering if anyone else has an idea that i could try to maybe fix this? Only reason why i ask here is because the Cherax support server does not appear to like having reports about co-loading.

**Comment by xiaoxiao921** [comment]
Duplicate of https://github.com/YimMenu/YimMenu/issues/2341#issuecomment-1778748488

**Reply by Bradly7412** [reply]
Cool thank you.
